reference/carts.v3.yml

```diff
@@ -1,4 +1,4 @@
-openapi: '3.0.3'
```

Comment:
this should retain single quotes. the datatype of the `openapi` value is `string`.

Reply:
@slsriehl I added the single quotes for `openapi` but I don't see where else I need to add them. Also, other files do not have single quotes for `openapi` which needs to be added.  I can create a ticket for these.